nautobot/extras/models/jobs.py

```diff
@@ -23,6 +23,7 @@
     setup_nautobot_job_logging,
 )
 from nautobot.core.celery.control import refresh_git_repository
+from nautobot.core.constants import CHARFIELD_MAX_LENGTH
```

Comment:
We should be able to deprecate the `JOB_MAX_GROUPING_LENGTH` constant and replace references to it with `CHARFIELD_MAX_LENGTH` throughout.

Reply:
So relate all places where `JOB_MAX_GROUPING_LENGTH` is used with `CHARFIELD_MAX_LENGTH` except for `ScheduledJob.task` right?

Reply:
`ScheduledJob.task` is using `JOB_MAX_NAME_LENGTH`, not `JOB_MAX_GROUPING_LENGTH`, and agreed, it should be left alone.